### DISCUSSION TITLE
Duplicated translation

### DISCUSSION THREAD

**Discussion by probonopd** [WeblateOrg/weblate#6686] [Q&A]
Getting

![image](https://user-images.githubusercontent.com/2480569/137623844-13d06bd6-3c85-4ca3-a29e-13060449d5e1.png)

https://hosted.weblate.org/projects/hellosystem/create-live-media/#alerts

These are my settings:

![image](https://user-images.githubusercontent.com/2480569/137623868-80ca96f3-9bda-4cb2-9b20-81ba087bf31a.png)

These are the ts files:

https://github.com/helloSystem/Utilities/tree/master/Utilities/Create%20Live%20Media.app/Resources/i18n

As you can see, there is only one _en.ts_ file which is generated from the source code to serve as the basis for the other languages.

What do I need to change to remove that error?

**Comment by nijel** [comment]
The alert should show detailed explanation, but it did not due to bug. I've fixed that in 2a4a0171a949ae5aa9d81779edacf011475fc5f7:

![image](https://user-images.githubusercontent.com/212189/142762997-362af718-0da3-4700-8c5f-b59d56eadea1.png)
